### ISSUE TITLE
ProjectK update through package manager removes properties from the project.json

### ISSUE THREAD

**Issue by zhili1208** [aspnet/Tooling#558]
Repro Step: 
File -> New Project -> Visual C# -> .Windows -> .NET Core Console Application (.NET COre)

You get a new project with a project.json which has type property in dependencies

"dependencies": {
 "Microsoft.NETCore.App": {
 "type": "platform",
 "version": "1.0.0-rc2-3002702"
 }

Then I wanted to upgrade Microsoft.NETCore.App to the latest private drop. I did this by right cliking on the 'References' node in the solution, -> Manage Nuget Packages. I have made a feed for the myget repo (if you don't already have it) 

https://dotnet.myget.org/F/dotnet-core/api/v3/index.json

and browsed to the Microsoft.NetCore.App package, and clicked the 'upgrade icon associated with that package. 

Once I upgraded, if I looked at my project.json I get 

"dependencies": {
 "Microsoft.NETCore.App": "1.0.0-rc4-000376"
 },

which did the upgrade but 'lost' the "type" : "platform" property

From https://github.com/NuGet/Home/issues/2888

**Comment by sayedihashimi** [comment]
TFS Bug#<a href="https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workItems#id=231149&fullScreen=false&_a=edit">231149</a>

**Comment by DaveVdE** [comment]
Yup this bit me this morning while updating to .NET Core 1.0.1. Any news?
